contractmanagement/src/main/java/com/elnido/modules/masterdata/controller/MaterialController.java

```diff
@@ -0,0 +1,96 @@
+package com.elnido.modules.masterdata.controller;
+
+import com.elnido.modules.masterdata.entity.Material;
+import com.elnido.modules.masterdata.entity.MaterialGroup;
+import com.elnido.modules.masterdata.model.MaterialPage;
+import com.elnido.modules.masterdata.service.MaterialGroupService;
+import com.elnido.modules.masterdata.service.MaterialService;
+import com.elnido.modules.masterdata.vo.MaterialSearchVO;
+import com.elnido.modules.masterdata.vo.MaterialVO;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.beanutils.BeanUtils;
+import org.jeecg.common.api.vo.Result;
+import org.jeecg.common.constant.CommonConstant;
+import org.springframework.web.bind.annotation.*;
+
+import javax.annotation.Resource;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author baogang
+ */
+@Slf4j
+@Api(tags = "物料维护")
+@RestController
+@RequestMapping("/masterdata/materials/")
```

Comment:
If we move slash to the begining of every method would that be batter.Can we avoid @PostMapping("")?